server/graphql/common/expenses.ts

```diff
@@ -1244,16 +1298,15 @@ export async function payExpense(req: express.Request, args: Record<string, unkn
         if (paypalPaymentMethod && paypalEmail === paypalPaymentMethod.name) {
           feesInHostCurrency['paymentProcessorFeeInHostCurrency'] = 0;
           await createTransactions(host, expense, feesInHostCurrency);
-        } else if (args.forceManual) {
+        } else if (forceManual) {
           await createTransactions(host, expense, feesInHostCurrency);
         } else if (paypalPaymentMethod) {
           return payExpenseWithPayPal(remoteUser, expense, host, paypalPaymentMethod, paypalEmail, feesInHostCurrency);
         } else {
           throw new Error('No Paypal account linked, please reconnect Paypal or pay manually');
         }
       } else if (payoutMethodType === PayoutMethodTypes.BANK_ACCOUNT) {
-        if (args.forceManual) {
-          feesInHostCurrency['paymentProcessorFeeInHostCurrency'] = 0;
```

Comment:
Removing this line fixes an issue introduced 2 years ago: we've been stripping out manually set payment processor fees.